schemas/styles/csl-names.rnc

```diff
@@ -0,0 +1,388 @@
+      font-formatting
+    }
+  
+  ## Inherits variable from the parent cs:names element.
+  names.label =
+    element cs:label {
+      [ a:defaultValue = "long" ] attribute form { term.form }?,
+      label.attributes-shared,
+      
+      ## When multiple='combined', set the delimiter for collapsed name labels
+      ## (e.g., ", " in "Creator, Writer, & Director")
+      [ a:defaultValue = ", " ] delimiter,
+      
+      ## Specify when the label delimiter is used between the second-to-last and last label of a label list;
+      ## only has an effect when the "and" term is used (e.g., "Writer & Director" or "Writer, & Director").
+      [ a:defaultValue = "contextual" ]
+      attribute delimiter-precedes-last {
+        
+        ## The label delimiter is only used when the label list consists of
+        ## three or more labels.
+        "contextual"
+        | 
+          ## The label delimiter is always used.
+          "always"
+        | 
+          ## The label delimiter is never used.
+          "never"
+      }?,
+      
+      ## Use to separate the second-to-last and last label of a label list by the "and" term.
+      attribute and {
+        
+        ## Use the "and" term "long" form (e.g., "Writer and Director").
+        "long"
+        | 
+          ## Use the "and" term "symbol" form (e.g., "Writer & Director").
+          "symbol"
+        | 
+          ## Use the "and" term "short" form. Not common in English, but is seen in other languages.
+          ## In German, for example: "Drehbuchautor u. Regisseur, where "u." is short for "und".
+          "short"
+      }?
+    }
+  
+  ## Render and format additional name components that are not otherwise shown;
+  ## components are rendered adjacent to their respective name in the name list (as with label).
+  names.component =
+    element cs:component { 
+
+      ## Which component to render?
+      attribute name {
+        
+        ## If name.form="long" (or name.form-subsequent="long" if a subsequent appearance of the name), render the "institution-short" name-part;
+        ## render nothing for other name forms.
+        "abbreviation"
+        |
+          ## Render the "alternate" form of a name;
+          ## name attributes are inherited from the cs:
+          "alternate"
+        |  
+          ## Render the "place" associated with the name (e.g. a publisher place).
+          "place"
+      },
+      affixes, 
+      font-formatting, 
+      strip-periods, 
+      text-case,
+
+      ## If name="alternate", specify the name form to render;
+      ## default to the values specified on the sibling cs:name
+      name.forms,
+
+      ## If name="alternate", specify formatting attributes for names;
+      ## default to the values specified on the sibling cs:name
+      name.attributes,
+
+      ## If name="alternate", use to apply separate formatting to individual name parts or to personal vs. institutional names 
+      name.name-part,
+      name.name-type
```

Comment:
```suggestion
      name.name-type }
```
I think this is the change you need for the one error, but then I get others.

Unless I'm doing something wrong with git, maybe double check the syntax problems and let me know when I can try again?